### ISSUE TITLE
Allow no delay

### ISSUE THREAD

**Issue by nloveladyallen** [fatiherikli/brainfuck-visualizer#35]
When running anything remotely complex, even the lowest delay setting slows down the program immensely. If I just want to see the end result of my program, I don't want to wait minutes for it to complete.

**Comment by shayakbanerjee99** [comment]
Yes, it would be great if there's a checkbox enabling which, the result is produce instantly without the memory operation animation.
